proxy/src/lib.rs

```diff
@@ -234,14 +244,15 @@ where
             let outgoing = Outbound::new(
                 bind,
                 control,
-                config.default_destination_namespace().to_owned(),
+                config.pod_namespace.to_owned(),
```

Comment:
This change shouldn't be part of this PR since it is out of scope.

Reply:
It was actually needed, because `IndexSet` is not `Copy`, and since one of is moved out of the `Config` earlier, it's no longer possible to call a method on `&Config`. Same with the `let report_timeout = ..` changes below.